netboot/netboot.go

```diff
@@ -46,6 +47,39 @@ func RequestNetbootv6(ifname string, timeout time.Duration, retries int, modifie
 	return conversation, nil
 }
 
+func RequestNetbootv4(ifname string, timeout time.Duration, retries int, modifiers ...dhcpv4.Modifier) ([]*dhcpv4.DHCPv4, error) {
+	var (
+		conversation []*dhcpv4.DHCPv4
+	)
+	delay := 2 * time.Second
+	for i := 0; i <= retries; i++ {
+		log.Printf("sending request, attempt #%d", i+1)
+		discover, err := dhcpv4.NewDiscoveryForInterface(ifname)
```

Comment:
Although this is consistent with what is being done in v6, I think it can be simplified by not creating the discover packet and passing nil to the `Exchange` call.

Reply:
Wait a sec, I might not be able to do that, as the discover I pass to `Exchange` has been modified with a `dhcpv4.WithNetboot` modifier.